.github/workflows/update-drci-comments.yml

```diff
@@ -0,0 +1,22 @@
+name: Update Dr. CI comments
+
+on:
+  schedule:
+    # Run every 15 minutes
+    - cron: "*/15 * * * *"
+  # Have the ability to trigger this job manually through the API (temporary)
+  workflow_dispatch:  
+
+jobs:
+  update-drci-comments:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Retrieve rockset query results and update Dr. CI comments
```

Comment:
Are we worried about running into the API rate limit if we're updating a ton of comments?

Reply:
Is there a way to estimate what our potential API usage might look like to help determine whether or not this would even be a feasible thing for us to do?

Reply:
I think the API rate limit is definitely a valid concern but I'm not sure what's the best way to address the issue. 

For each PR there are either 1 or 2 REST api calls -- the first is to get the current comment ID and check if the body of the comment is up to date, and the second call is to update the comment if the body is out of date. These calls happen for all open PRs on trunk that have any workflow check that's been completed in the last 15 minutes, which on average seems to be around 10-15 PRs at a time. So with this current method there would be 20-30 rest api calls every 15 min.

Reply:
Okay I think with that we should probably be fine in regards if the volume is that low